src/app/app.component.ts

```diff
@@ -22,6 +23,7 @@ export class App {
       .then(() => {
         this.configureStatusBar();
         this.loadAppInfo();
+        this.loadAirwatchConfig();
```

Comment:
Could this cause a race condition? We need this config endpoint immediately after log in, maybe it should be in `login.ts`?

Reply:
Have moved it to login where we dispatch the action and subscribe to the results passing them into the login function